fgpyo/sam/__init__.py

```diff
@@ -547,6 +588,46 @@ def length_on_target(self) -> int:
         """Returns the length of the alignment on the target sequence."""
         return sum([elem.length_on_target for elem in self.elements])
 
+    def query_alignment_offsets(self, reverse: bool = False) -> Optional[RangeOfBases]:
+        """Gets the 0-based, end-exclusive positions of the first and last aligned base in the
+        query. The resulting range will contain the range of positions in the SEQ string for
+        the bases that are aligned. If no bases are aligned, the return value will be None.
```

Comment:
```suggestion
        """
        Gets the 0-based, end-exclusive positions of the first and last aligned base in the query.
        
        The resulting range will contain the range of positions in the SEQ string for
        the bases that are aligned. If no bases are aligned, the return value will be None.
```

Reply:
Suggested change is to adhere to Google style guidelines of a single summary sentence separated from the longer description.

Reply:
done.